examples/basics/visuals/grid_mesh.py

```diff
@@ -0,0 +1,53 @@
+"""
+Demonstration of a mesh constructed in a grid about x,y,z coordinates.
+"""
```

Comment:
argh, I thought we had better documentation than that. Don't worry about it for now, then. Sorry for the noise.